### DISCUSSION TITLE
[Question]: what per centage change of field of view would necessitate a new model?

### DISCUSSION THREAD

**Discussion by ElectronicBattle** [blakeblackshear/frigate#13404] [Ask A Question] [answered]
### What is your question?

Hello all

I have finally submitted a model request, realising I needn't be as cautious as I have been. Purely to satisfy my interest, how much of a change in a camera's position would mean a new model - is it something which users have experienced and can comment on please?

For example:
this
![image](https://github.com/user-attachments/assets/6becc2d9-838d-44b0-88e9-55bd0307a698)
changing to this
![image](https://github.com/user-attachments/assets/da416385-ad1b-47c9-996f-614c9917850b)
probably doesn't seem much of a change to the human eye but this
![image](https://github.com/user-attachments/assets/68955536-6651-4778-a08a-dc0371957a6d)
vs this
![image](https://github.com/user-attachments/assets/9e327a2b-c3cd-4be6-8723-8476d5b2fdd2)
seems pretty significant a change.

(The whole image in this case is here:)
![image](https://github.com/user-attachments/assets/81bc7697-ea5b-4f9b-a8e9-cb46a562948a)

Furthermore, I imagine that  changing from one resolution of CCD sensor (1MP) to another (2MP or 4MP etc.) would require a model change but what if you went from a sensor of one size (1") to another (7/8") but with the same resolution?

**Comment by hawkeye217** [comment, marked as answer]
Users have reported that even after changing their field of view or adjusting the angle on a particular camera, there's been no change in detection performance from Frigate+ models. I don't suspect a resolution change will cause any issues, either. So I don't think you have anything to worry about.

**Reply by ElectronicBattle** [reply]
very interesting; the superficial understanding of MD and object classification which I thought I had is clearly significantly flawed!

**Reply by ElectronicBattle** [reply]
As an aside - after a couple of hours at the most since submitting my request, my model's ready!
